### ISSUE TITLE
mac version

### ISSUE THREAD

**Issue by aliafshany** [TheGreenPig/nordvpn-account-checker#1]
hi there
what do you suggest to get it work on mac?
best

**Comment by SS-Einsatzgruppe** [comment]
Bruh....

NordVPN Account and Connection Checker

---->ONLY WORKS ON LINUX! <----------

A script that takes a .txt file as a password Inupt and tries to login and then connect to a server. If the account information is wrong or the connection fails, the script just moves on to the next entry.

**Comment by TheGreenPig** [comment]
Yes the script uses the subprocess.run Function and the terminal so I am pretty sure it doesn't work on any OS besides Linux. I wouldn't know a way how to make it work for Windows or mac either. Like I said in the README though, I am fairly new to Python and Linux in general so I might be missing something and there is a way in fact how to make it work. If anyone has an idea how to do so I would be happy to hear!

**Comment by aliafshany** [comment]
aha, thank you for replying, then it seems i gotta get it work on linux
👍
